index.html

```diff
@@ -671,10 +671,241 @@
+					so this constraint on styling is easily worked around
+					when subdivisions of a publication need to be blended together.</p>
 
-				<p class="ednote">Placeholder for paginated reading experience.</p>
+				</div>
+
+					<div class=issue>
+						<p>Despite this general requirement that each resource should be treated as a separate document
+						for the purpose of layout,
+						there are probably a handful of places where CSS specifications should be amended to
+						be able to deal more intelligently with collections of resources like <a>Web Publications</a>.
+
+						<p>One instance is the definition of <a href="https://drafts.csswg.org/css-content/#cross-references">cross references</a>
+						which are currently restricted to work only within a single document.
+						This restriction should be relaxed to allow for cross references between
+						separate resources of a single <a>Web Publication</a>.
+
+						<p>Another related one would be to allow <a href="https://drafts.csswg.org/css-lists-3/#counter-properties">counters</a>
+						to accumulate across multiple resources of a single <a>Web Publication</a>,
+						for instance so that figures in multiple sections may be numbered in a single sequance.
+
+					</div>
+
+				<section id="scrolling-or-paginating" class=informative>
+					<h4>Scrolling or Paginating</h4>
+					<p>Web content, and a fortiori <a>Web Publications</a>, may be rendered on various media,
+					each with differing characteristics,
+					either due to the physical media itself or to the design of the User Agent.
+					Web Publications are not constrained to any specific medium;
+					User Agents are expected to render them as they would render other web content on the same medium.
+					Authors can adjust the way they style their documents using the features specified in [[MEDIAQ]] or its successor [[MEDIAQUERIES-4]],
+					as supported by the User Agent.</p>
+					<p>It is however worth calling out a major subdivision within media types: scrolling and paginating.</p>
+					<p>In scrolling media, when content overflows the <a href="https://www.w3.org/TR/css-display-3/#initial-containing-block">initial containing block</a> (See [[CSS-DISPLAY-3]])
+					it is exposed by allowing users to scroll to it.
+					This behavior is typical of most desktop or mobile web browsers.</p>
+					<p>In paginating media, content is broken up into discrete pages,
+					and content that overflows one page is displayed on subsequent pages.
+					This behavior is typical of web content printed on paper,
+					of the output of web-to-PDF renderers,
+					and of many e-book reading systems.</p>
+					<p>Some User Agents may support both scrolling and pagination.
+					As the preferences of individual readers vary,
+					and as different types of publications may be better suited for one or the other,
+					this specification encourages User Agents to support both,
+					and to offer a choice to their users.</p>
```

Comment:
it also needs to be an attribute of a publication as a given author may have a clear position on how they wish their content to be presented